src/uct/api/uct_def.h

```diff
@@ -309,6 +315,11 @@ typedef struct uct_cm_listener_conn_request_args {
      * Remote data from the client.
      */
     const uct_cm_remote_data_t *remote_data;
+
+    /**
+     * Client's address.
+     */
+    ucs_sock_addr_t             *client_sockaddr;
```

Comment:
1. rename it to "client_addr"
2. make it just ucs_sock_addr_t without a pointer

Reply:
maybe even "client_address"